.github/workflows/build_windows.yml

```diff
@@ -11,19 +11,19 @@ jobs:
       matrix:
         python-version: [ '3.6', '3.7', '3.8' ]
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
```

Comment:
you're using v1 everywhere else, right? Is this an upgrade step-by-step, or are some workflows incompatible with v2?

Reply:
I need to upgrade everything to 2.0

Reply:
Done